modules/social_features/social_user_export/social_user_export.module

```diff
@@ -85,6 +85,65 @@ function _social_user_export_get_view(array $query = [], $execute = TRUE) {
   return $view;
 }
 
+/**
+ * Returns username of user.
+ *
+ * @param \Drupal\user\UserInterface $account
+ *   The account to get the data for.
+ *
+ * @return mixed
+ *   The username of the user.
+ */
+function social_user_export_user_name(UserInterface $account) {
+  $query = \Drupal::database()->select('users_field_data', 'ufd');
```

Comment:
Please use `$account->getAccountName()` instead (defined on the AccountInterface interface and implemented in the User entity).

No database query is needed for this.